src/modules/telemetry/i18n/nl_NL.root.json

```diff
@@ -14,8 +14,14 @@
 	"info": {
 		"close": "Toon deze melding niet overnieuw",
 		"decline": "Sta het verzamelen van metadata niet toe",
-		"text": "De LSS Manager verzamelt metadata zoals beschreven in de <a href='{wiki}metadata.html' class='lightbox-open'>Wiki</a>.",
+		"text": "De LSS Manager verzamelt metadata zoals beschreven in de <a href='{wiki}metadata.html' class='lightbox-open'>Wiki</a>.<br>U kunt uw akkoord op een later moment altijd herroepen.",
```

Comment:
@JRH-1997 @Piet2001 is this good nl_NL translation?

Reply:
U kunt uw akkoord op een later moment altijd herroepen.
to:
U kunt uw toestemming ten alle tijden intrekken.

Reply:
@Tsumiki-Chan ;)